src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-table-body-cell.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+  import WorkflowStatus from '$lib/components/workflow-status.svelte';
+  import FilterOrCopyButtons from '$lib/holocene/filter-or-copy-buttons.svelte';
+  import { workflowFilters, workflowSorts } from '$lib/stores/filters';
+  import { timeFormat } from '$lib/stores/time-format';
+  import {
+    isDataCell,
+    isPathCell,
+    type WorkflowHeader,
+    WORKFLOW_CELLS,
+  } from '$lib/stores/workflow-table-columns';
+  import { updateQueryParamsFromFilter } from '$lib/utilities/query/to-list-workflow-filters';
+
+  export let column: WorkflowHeader;
+  export let workflow: WorkflowExecution;
+
+  $: ({ label } = column);
+
+  let cellContent: string;
+  let filterOrCopyButtonsVisible: boolean = false;
+
+  const showFilterOrCopy = () => (filterOrCopyButtonsVisible = true);
+  const hideFilterOrCopy = () => (filterOrCopyButtonsVisible = false);
+
+  const onRowFilterClick = (header: WorkflowHeader, value: string) => {
+    const attribute =
+      header.label === 'Workflow ID' ? 'WorkflowId' : 'WorkflowType';
+    const filter = $workflowFilters.find((f) => f.attribute === attribute);
+    const getOtherFilters = () =>
+      $workflowFilters.filter((f) => f.attribute !== attribute);
+
+    if (!filter) {
+      const newFilter = {
+        attribute,
+        value,
+        conditional: '=',
+        operator: '',
+        parenthesis: '',
+      };
+      $workflowFilters = [...getOtherFilters(), newFilter];
+    } else {
+      $workflowFilters = [...getOtherFilters()];
+    }
+
+    updateQueryParamsFromFilter($page.url, $workflowFilters, $workflowSorts);
+  };
+
+  $: cell =
+    label in WORKFLOW_CELLS ? WORKFLOW_CELLS[label] : { label, path: label };
+  $: {
+    if (isPathCell(cell)) {
+      cellContent = workflow[cell.path] ?? '';
+    } else if (isDataCell(cell)) {
+      cellContent = cell.data(workflow, $timeFormat) ?? '';
+    }
+  }
+</script>
+
+{#if label === 'Status'}
+  <td class="workflows-summary-table-body-cell">
+    <WorkflowStatus status={workflow.status} />
+  </td>
+{:else if label === 'Type' || label === 'Workflow ID'}
+  <td
+    class="workflows-summary-table-body-cell relative"
+    on:mouseover={showFilterOrCopy}
+    on:focus={showFilterOrCopy}
+    on:mouseleave={hideFilterOrCopy}
+    on:blur={hideFilterOrCopy}
+  >
+    <span class="table-link">{cellContent}</span>
+    <FilterOrCopyButtons
```

Comment:
These are easily hidden from view in pinned columns... Wondering if we should put them at the beginning of the table cell instead? 

![Screenshot 2023-03-27 at 3 53 24 PM](https://user-images.githubusercontent.com/15069288/228078555-d72ad296-7877-4655-8cab-93b07b8ca189.png)
![Screenshot 2023-03-27 at 3 53 38 PM](https://user-images.githubusercontent.com/15069288/228078565-c08cdc4f-ef3c-49e7-8e0b-95b7e3810c4a.png)

Reply:
I'm still not totally sold on the resizable columns. If you wanted more space to see the other columns, shouldn't you remove a pinned column? Like in the above case, move Workflow ID out of the pinned columns.

Reply:
Yeah, I think the resizable pinned section might be more complicated than it's worth. Removing it 👍

Reply:
removed in [296e2d6](https://github.com/temporalio/ui/pull/1229/commits/296e2d663ea227a8f2918439e86c6deb0b06a9e3)